Add example services

smarlowucf opened the pull request:
- Add plugin_key option to config
- Convert spec to class based to allow for single module plugins
- Move plugin_manager instance to an importable namespace
- Rename job_config arg in spec to data to be more generic

smarlowucf committed:
Add example services

- Add plugin_key option to config
- Convert spec to class based to allow for single module plugins
- Move plugin_manager instance to an importable namespace
- Rename job_config arg in spec to data to be more generic

apozsuse commented on examples/wx.py:
```diff
@@ -0,0 +1,52 @@
+    def run_task(self, data, log_callback):
+        """Get current Wx for given location"""
+        data['wx_data'] = {
+            'Temp': '15C',
```
May be changing the values in both examples to be more visual which one is executed?

smarlowucf committed:
Change the example wx data values

Make the forecast and current values different to make it easier
to visually distinguish.
